<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_play</class>
 <widget class="QDialog" name="user_play">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>69</y>
     <width>821</width>
     <height>621</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="HOME">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>71</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/football.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="GW">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 24pt &quot;Segoe UI&quot;;
color: rgb(55, 0, 60);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="Points">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 24pt &quot;Segoe UI&quot;;
color: rgb(55, 0, 60);</string>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="TOTAL_POINTS">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>330</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 24pt &quot;Segoe UI&quot;;
color: rgb(55, 0, 60);</string>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="p2_22">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>300</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/athlete.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p1_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>591</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/football-field (1).png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p2_3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>170</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/athlete.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p2_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>300</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/athlete.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p2_5">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>400</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/athlete.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p2_6">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>170</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/athlete.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="p4">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>230</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="p5">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>230</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="p3">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>370</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="p2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>370</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="p1">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>470</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="GW_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 24pt &quot;Segoe UI&quot;;
color: rgb(55, 0, 60);</string>
     </property>
     <property name="text">
      <string>GW</string>
     </property>
    </widget>
    <widget class="QLabel" name="team_name">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 24pt &quot;Segoe UI&quot;;
color: rgb(55, 0, 60);</string>
     </property>
     <property name="text">
      <string>GW</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>370</y>
       <width>71</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/football.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>70</y>
       <width>71</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/football.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>280</y>
       <width>71</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/football.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>71</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/football.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>360</y>
       <width>71</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/football.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>71</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/football.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="GW_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 24pt &quot;Segoe UI&quot;;
color: rgb(55, 0, 60);</string>
     </property>
     <property name="text">
      <string>POINTS</string>
     </property>
    </widget>
    <widget class="QLabel" name="GW_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 24pt &quot;Segoe UI&quot;;
color: rgb(55, 0, 60);</string>
     </property>
     <property name="text">
      <string>Over All</string>
     </property>
    </widget>
    <zorder>label_10</zorder>
    <zorder>label_7</zorder>
    <zorder>p1_2</zorder>
    <zorder>label_2</zorder>
    <zorder>GW</zorder>
    <zorder>Points</zorder>
    <zorder>TOTAL_POINTS</zorder>
    <zorder>p2_22</zorder>
    <zorder>p2_3</zorder>
    <zorder>p2_4</zorder>
    <zorder>p2_5</zorder>
    <zorder>p2_6</zorder>
    <zorder>p4</zorder>
    <zorder>p5</zorder>
    <zorder>p3</zorder>
    <zorder>p2</zorder>
    <zorder>p1</zorder>
    <zorder>GW_2</zorder>
    <zorder>team_name</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_11</zorder>
    <zorder>label_16</zorder>
    <zorder>GW_3</zorder>
    <zorder>GW_4</zorder>
   </widget>
   <widget class="QWidget" name="TRMARKET">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>TSMARKET</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>350</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>STANDing</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Home">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>121</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton#Home
{

	
	background-color: rgb(255, 255, 255);
 
	image: url(:/img/img/home.png);
}


QPushButton#Home:hover
{


   font: 700 10pt &quot;Segoe UI&quot;;
	background-color:  rgb(37, 255, 26);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="TSM">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>121</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton#TSM
{

	
	background-color: rgb(255, 255, 255);
 
	
	image: url(:/img/img/left-right.png);
}


QPushButton#TSM:hover
{


   font: 700 10pt &quot;Segoe UI&quot;;
	background-color:  rgb(37, 255, 26);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="STANDING">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>121</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton#STANDING
{

	
	background-color: rgb(255, 255, 255);
 
	
	image: url(:/img/img/seo.png);
}


QPushButton#STANDING:hover
{


   font: 700 10pt &quot;Segoe UI&quot;;
	background-color:  rgb(37, 255, 26);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>20</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/img/logo_transparent.png);
background-color: rgb(55, 0, 60);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>20</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>20</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>20</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>20</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>20</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>20</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>791</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(55, 0, 60);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>20</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>20</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>20</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>20</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>label</zorder>
  <zorder>Home</zorder>
  <zorder>TSM</zorder>
  <zorder>STANDING</zorder>
  <zorder>label_19</zorder>
  <zorder>label_8</zorder>
  <zorder>label_12</zorder>
  <zorder>label_17</zorder>
  <zorder>label_15</zorder>
  <zorder>label_14</zorder>
  <zorder>label_18</zorder>
  <zorder>label_23</zorder>
  <zorder>label_22</zorder>
  <zorder>label_20</zorder>
  <zorder>label_21</zorder>
  <zorder>label_13</zorder>
  <zorder>label_9</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
